src/m365/todo/commands/task/task-add.ts

```diff
@@ -64,26 +118,44 @@ class TodoTaskAddCommand extends GraphCommand {
     try {
       const listId: string = await this.getTodoListId(args);
 
-      const requestOptions: any = {
+      const requestOptions: AxiosRequestConfig = {
         url: `${endpoint}/me/todo/lists/${listId}/tasks`,
         headers: {
           accept: 'application/json;odata.metadata=none',
           'Content-Type': 'application/json'
         },
         data: {
-          title: args.options.title
+          title: args.options.title,
+          body: {
+            content: args.options.bodyContent,
+            contentType: args.options.bodyContentType?.toLowerCase() || 'text'
+          },
+          importance: args.options.importance?.toLowerCase(),
+          dueDateTime: this.getDateTimeTimeZone(args.options.dueDateTime),
```

Comment:
dueDateTime doesn't seem to take time into consideration. See image below. 

![afbeelding](https://user-images.githubusercontent.com/38426621/196458129-5217aaea-139e-4923-9ee9-38faa41e7429.png)